example/web-avatar-client/test/test-data.ts

```diff
@@ -0,0 +1,108 @@
+export const validMCharacter = `
```

Comment:
Would be good to include syntactically-invalid strings too. Even if the `DOMParser` implementation handles this simply it's good to have defined behaviour in case the implementation changes.

Reply:
@MarcusLongmuir Can you give me one example of such strings?